libpromises/evalfunction.c

```diff
@@ -4209,6 +4209,23 @@ static FnCallResult FnCallShuffle(EvalContext *ctx, ARG_UNUSED const Policy *pol
     return (FnCallResult) { FNCALL_SUCCESS, (Rval) { shuffled, RVAL_TYPE_LIST } };
 }
 
+static FnCallResult FnCallInt(ARG_UNUSED EvalContext *ctx, ARG_UNUSED const Policy *policy, ARG_UNUSED const FnCall *fp, const Rlist *finalargs)
+{
+    if(finalargs == NULL)
+    {
+        return FnFailure();
```

Comment:
This should also log an error, I suppose? Or, if this is actually handled by the parser, just replace the `if` with an `assert()`.